verifier/verifier.go

```diff
@@ -518,6 +545,67 @@ func verifyAuthenticTimestamp(outcome *notation.VerificationOutcome) *notation.V
 	}
 }
 
+func verifyRevocation(outcome *notation.VerificationOutcome, client *http.Client, logger log.Logger) *notation.ValidationResult {
+	r, err := revocation.New(client)
+	if err != nil {
+		logger.Debug("no HTTP client provided for revocation checking")
```

Comment:
In future there can be other reasons so its better to get reason from `err`
```suggestion
		logger.Debug("error while calling revocation.New: %s , err.Error())
```